shard.lua

```diff
@@ -810,41 +831,31 @@ local function get_server_list()
     return configuration.servers
 end
 
-local function is_shard_connected(uri, conn)
```

Comment:
Why you've deleted `is_shard_connected`? Where is it now or what is used instead of it.

Reply:
a better function `net_connect` is introduced. is_shard_connected was used only in `shard_connect`. Basically what I did is rewritten connect mechanism.